app/routes.py

```diff
@@ -0,0 +1,38 @@
+from app import app
+from app.blacklist import blacklist
```

Comment:
Why did you import `blacklist` here?

I don't see it being used anywhere in this file

Reply:
done